src/centreon/plugins/passwordmgr/centreonvault.pm

```diff
@@ -25,28 +25,41 @@ use warnings;
 use Data::Dumper;
 use centreon::plugins::http;
 use JSON::XS;
+use MIME::Base64;
+use Crypt::OpenSSL::AES;
+use centreon::plugins::statefile;
+
+my $VAULT_PATH_REGEX = qr/^secret::hashicorp_vault::([^:]+)::(.+)$/;
 
 sub new {
     my ($class, %options) = @_;
     my $self  = {};
     bless $self, $class;
 
     if (!defined($options{output})) {
-        print "Class PasswordMgr: Need to specify 'output' argument.\n";
+        print "Class PasswordMgr: Needs to be passed an 'output' argument of class output.\n";
```

Comment:
The error message is not clear

Reply:
This message will appear only if a developer changes the way password managers are called here https://github.com/centreon/centreon-plugins/blob/056589f360e52434923a86456b6df9651666d34d/src/centreon/plugins/script_custom.pm#L177. 
It shouldn't ever happen in production